src/builder.rs

```diff
@@ -1016,50 +1280,50 @@ impl<V> Bundle<InProgress<Proof, Unauthorized>, V> {
     }
 }
 
-impl<P: fmt::Debug, V> Bundle<InProgress<P, PartiallyAuthorized>, V> {
+impl<Proof: fmt::Debug, V, P: OrchardPrimitives>
+    Bundle<InProgress<Proof, PartiallyAuthorized>, V, P>
```

Comment:
_From str4d ([r2602938883](https://github.com/zcash/orchard/pull/471#discussion_r2602938883))_

Ditto:
```suggestion
impl<P: fmt::Debug, V, Pr: OrchardPrimitives>
    Bundle<InProgress<P, PartiallyAuthorized>, V, Pr>
```

Reply:
FIXED